assets/js/main.js

```diff
@@ -0,0 +1,323 @@
+          HumanPlayerCardsArr[index].onclick = function (e) {
+            var humanPlayed = playCard(Human, parseInt(e.target.id.substring(2,e.target.id.length))) //input: player & card clicked
+            if (humanPlayed && (DoNotSwitchTurn === false)) {
+              var computerPlayed = playCard(Computer,null)
+
+            } else {
+              //GameMessage.innerHTML = 'Human Player, awaiting your play...'
+            }
+            return false
+          }
+        })
+
+    // Presents Computer Player's Pile
+    ComputerPlayerPile.forEach(function (card) {
+          var listCard = document.createElement('li')
+          listCard.textContent = card.label
+          listComputerPlayerPile.appendChild(listCard)
+        })
+
+    // Presents opening card from Draw Pile
+    DiscardPile.forEach(function (card) {
+          var listCard = document.createElement('li')
+          listCard.textContent = card.label
+          listDiscardPile.appendChild(listCard)
+        })
+
+    // unlist intial DrawPile
+    while (listDrawPile.hasChildNodes()) {
+      listDrawPile.removeChild(listDrawPile.firstChild);
+    }
+
+    // relist shuffled DrawPile with dealt & opening cards removed
+    DrawPile.forEach(function (card) {
+          var listCard = document.createElement('li')
+          listCard.textContent = card.label
+          listDrawPile.appendChild(listCard)
+        })
+
+    GameStarted = true
+
+    window.setTimeout(function () {
+      GameMessage.innerHTML = 'Human Player, awaiting your play...'
+    }, 500)
+  }
+} // end of startGame
+
+function resetGame () {
+  GameStarted = false
+  window.location.reload(true)
+}
+
+function clearPileUX (whichPile) {
+  if (whichPile === Human) {
+    while (listHumanPlayerPile.hasChildNodes()) {
+      listHumanPlayerPile.removeChild(listHumanPlayerPile.firstChild)
+    }
+  } else if (whichPile === Computer) {
+    while (listComputerPlayerPile.hasChildNodes()) {
+      listComputerPlayerPile.removeChild(listComputerPlayerPile.firstChild)
+    }
+  } else if (whichPile === Discard) {
+    while (listDiscardPile.hasChildNodes()) {
+      listDiscardPile.removeChild(listDiscardPile.firstChild)
+    }
+  } else if (whichPile === Draw) {
+    while (listDrawPile.hasChildNodes()) {
+      listDrawPile.removeChild(listDrawPile.firstChild)
+    }
+  }
+}
+
+function displayPileUX (whichPile) {
+  if (whichPile === Human) {
+    HumanPlayerPile.forEach(function (card, index) {
+          var listCard = document.createElement('li')
+          listCard.innerHTML = '<a id="hp'+index+'" href="#">'+card.label+'</a>'  //let listed card be clickable
+          listHumanPlayerPile.appendChild(listCard)
+          HumanPlayerCardsArr[index] = document.querySelector('#hp'+index)
+          HumanPlayerCardsArr[index].onclick = function (e) {
+            var humanPlayed = playCard(Human, parseInt(e.target.id.substring(2,e.target.id.length))) //input: player & card clicked

```

Comment:
what does `e.target.id.substring(2,e.target.id.length))`, that's quite a long code that you should make variable instead